website/docs/starrocks.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 8
```

Comment:
You also need to modify the sidebars.js file to actually see the option to select the StarRocks query engine on the site. Refer to the readme [here](https://github.com/onetable-io/onetable/tree/main/website#docs).